kimchi_state/include/kimchi_state/navigation_manager.h

```diff
@@ -6,7 +6,7 @@
 #include <queue>
 #include <rclcpp/rclcpp.hpp>
 
-#include "kimchi_navigation/global_localization.hpp"
+#include "kimchi_interfaces/action/localizing.hpp"
```

Comment:
Reasoning: global localization is not needed here. This just needs the localizing action